crates/metrics/src/main.rs

```diff
@@ -125,12 +179,51 @@ fn print_cloudwatch_stats(
     stats.write_csv(std::io::stdout()).unwrap();
 }
 
-fn print_log_stats(log_file: String) {
+fn print_cloudwatch_metrics(
+    query_args: &QueryArgs,
+    log_group_name: String,
+    region: &Region,
+    assume_role_arn: &str,
+) {
+    let cloudwatch = CloudWatchLogger::with_log_group(
+        log_group_name,
+        crate::cloudwatch::assume_role(&region, assume_role_arn),
+        region,
+    );
+
+    let metrics = cloudwatch.query_metrics(query_args);
+    let file = BufWriter::new(std::io::stdout());
+    let mut writer = csv::Writer::from_writer(file);
+
+    for m in metrics {
+        writer.serialize(m).unwrap();
+    }
+    writer.flush().unwrap();
+}
+
+// TODO use aggregation patern
```

Comment:
TODO for this PR, or later?

Reply:
Later is fine, this is for generating metrics from just local log files, not cloudwatch.